src/CommunityToolkit.Maui/Essentials/AppTheme/AppThemeColor.shared.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CommunityToolkit.Maui;
+
+/// <summary>
+/// Represents a color that is aware of the operating system theme.
+/// </summary>
+public sealed class AppThemeColor : AppThemeObject<Color>
```

Comment:
Shall we rename this to `ThemeColor`?

After renaming `ThemeResourceExtension`, it's a bit confusing to be using `<mct:AppThemeColor ...`, `<mct:AppThemeResource ...`, and `{mct:ThemeResource ...` in XAML. 

I assume devs would prefer consistency of each being named `Theme*`